.gitignore

```diff
@@ -321,4 +321,4 @@ share/
 /modules/reactor/unit/reactor-unit.yaml
 /modules/rdg/unit/rdg-unit.yaml
 /modules/fsi/unit/fsi-unit.yaml
-/tutorials/tutorial01_app_development/**/babbler.yaml
+/tutorials/**/*.yaml
```

Comment:
```suggestion
/tutorials/darcy_thermo_mech/*/darcy_thermo_mech.yaml
```

Reply:
This change would then not cover the `babbler.yaml` files, when I want this line to cover both. I will modify this to just be more explicit across both tutorials, since that seems to be your preference here.